lib/core/src/msParam.cpp

```diff
@@ -1022,6 +1022,28 @@ parseMspForPosInt( msParam_t* inpParam ) {
     return myInt;
 }
 
+int parseMspForStr_new(msParam_t* inpParam, char** outParam)
```

Comment:
Here's some suggestions for naming (ordered). _You're not obligated to use them._
- `msp_to_string` _(This would be nice if all parsing functions had the `msp_` prefix)_
- `parse_msparam_as_string`
- `parse_msp_as_string`
- `parse_msp_for_string` _(yuck)_

The first one fits the namespacing scheme used in many C libs.

If there are better names, please list them.

Reply:
I like the first one, too - there would be a family of these (`msp_to_*`)

The second one also works...

Perhaps a combination?   `parse_msparam_to_string` or `msparam_to_string` ?

Reply:
1 looks pretty nice. Feels nice and orderly to have a family of happy functions.